README.md

```diff
@@ -1,251 +1,133 @@
-    - row: change number of vertex of the named label
-- EdgeTypes: contains any number of different vertex labels
-  - name: change the first edgelabel name
-  - columns: may contain any number of columns(edge property keys)
-    - [String]: Name of column
-    - dataType: String, Long, Integer, and Date.
-    - dateRange: default from 1970 to current time in default "dd-MMM-yyyy" format
-      unless specified in the date Format
-    - dateFormat: support JAVA SimpleDateFormat patterns. Default: "dd-MMM-yyyy"
-    - composit: true or false
-    - mixedIndex: any String (default should be "search") 
-- relations: Defines how the named edge relations can happen and number of occurrences.
-  - left: the outV of an edge
-  - right: the inV of an edge
-  - supernode: nodes have many edges
-    - vertices: n where n is the first n nodes of left
-    - edges: additional edges are added to each of the n nodes
-    
-### Import CSV file to JanusGraph
-JanusGraphBatchImporter is a tool that imports bulk data into JanusGraph using multiple CSV files for the data and json config files to define the schema and mapping between schema and data. It uses multiple workers in order to achieve full speed of import of the data.
-
-JanusGraphBatchImporter depends on JanusGraph libraries and also commons-csv-1.4.jar
-
-In order to use under Linux:
-- Edit conf/batch_import.properties file to configure the importer
-- Usage:  
-  `run.sh import <janusgraph-config-file> <data-files-directory> <schema.json> <data-mapping.json>`
-  - `<janusgraph-config-file>`: Properties file with the JanuGraph configuration
-  - `<data-files-directory>:` Relative directory where the data files are located
-  - `<schema.json>:` JSON file defining the schema of the graph
-  - `<data-mapping.json>`: mapping file defining the relationship between the CSV/s fields and the graph
-
-- Example:  
-  ```
-  ./run.sh import /root/janusgraph-v0.1.1/conf/janusgraph-cassandra-es.properties \
-      /tmp /tmp/schema.json /tmp/datamapper.json
-  ```
-- Using different JanusGraph lib  
-  ```
-  export JANUSGRAPH_HOME=/path-to-your-janusgraph-home  
-  ./run.sh import /root/path-to-your-janusgraph-home/conf/janusgraph-cassandra-es.properties \
-      /tmp /tmp/schema.json /tmp/datamapper.json
-  ```
+cp ~/janusgraph-utils/samples/date-helper.groovy ../../scripts 
+cp ../janusgraph-cql-es.properties janusgraph-cql-es-server.properties
+Add a line to janusgraph-cql-es-server.properties, gremlin.graph=org.janusgraph.core.JanusGraphFactory
+cp gremlin-server.yaml rest-gremlin-server.yaml
+Change rest-gremlin-server.yaml
+host: x.x.x.x (your server ip)
+channelizer: org.apache.tinkerpop.gremlin.server.channel.HttpChannelizer
+graph: conf/gremlin-server/janusgraph-cql-es-server.properties}
+scripts: [scripts/empty-sample.groovy,scripts/date-helper.groovy]}}
+cd ~/janusgraph; ./bin/gremlin-server.sh ./conf/gremlin-server/rest-gremlin-server.yaml
+```
+Query and update graph data using REST
```

Comment:
Looks like this sentence needs a period especially because this and the one after end up on the same line smashed together.